packages/signature-controller/src/SignatureController.ts

```diff
@@ -40,7 +40,6 @@ import {
   PersonalMessageManager,
   TypedMessageManager,
 } from '@metamask/message-manager';
-import { providerErrors } from '@metamask/rpc-errors';
```

Comment:
Is there anymore references to this package or can we remove `@metamask/rpc-errors` from `package.json`?

Reply:
done